debian/control

```diff
@@ -220,6 +220,7 @@ Depends: ceph-base (= ${binary:Version}),
          python3-pecan,
          python3-requests,
          python3-werkzeug,
+         python3-distutils,
```

Comment:
nit, could you move this up before `python3-jwt,` the they are ordered alphabetically?

Reply:
Thanks for the pointer, I somehow missed this completely. I adjusted the order.